module/userProjects/helper.js

```diff
@@ -490,6 +490,73 @@ module.exports = class UserProjectsHelper {
                     result["link"] = solutionAndProgramCreation.data.parentSolutionInformation.link ? solutionAndProgramCreation.data.parentSolutionInformation.link : "";
                 }
 
+                return resolve({
+                    success: true,
+                    data: result
+                });
+
+            } catch (error) {
+                return resolve({
+                    status:
+                        error.status ?
+                            error.status : HTTP_STATUS_CODE['internal_server_error'].status,
+                    success: false,
+                    message: error.message,
+                    data: {}
+                });
+            }
+        });
+    }
+
+     /**
```

Comment:
@VISHNUDAS-tunerlabs can you add request and response here too for better clarity

Reply:
@ankitpws added